### ISSUE TITLE
About the learning rate

### ISSUE THREAD

**Issue by LWHYC** [Haochen-Wang409/U2PL#83]
Hi! Thank you for the contribution!
I'm running the semi experiments on 4 A100(80GB) GPU and set the batchsize on each GPU to 4.
So I suppose I should set the learning rate to 0.005?

Thanks.

**Comment by Haochen-Wang409** [comment]
If you plan to train a semi-supervised model, the learning rate should be set to 0.001 on VOC 2012 and 0.01 on Cityscapes, respectively, since the ```batch_size``` equals to 16 (4x4), which is the same as the ```batch_size``` in ```config.yaml``` (*i.e.*, 4x4 for VOC 2012 and 2x8 for Cityscapes).
